install.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+# Create directories
+mkdir -p /tmp/tune-bot
+mkdir -p library
+mkdir -p bin
+
+# Install dependencies
+apt update
+apt upgrade -y
+apt install -y curl git mysql-server python-is-python3
```

Comment:
Although you may have noticed some duplication, I put the dependencies separately into their own install scripts so that each one could be run independently without needing any other scripts to have run first. The idea was if one day each item was running on individual servers, it would be easy to just separate it out.

Reply:
I see, I can put them back